packages/amazonq/.changes/next-release/Feature-adb7e3c8-9c3e-4090-a283-2d29fc396f04.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "feat(Amazon Q Code Transformation): warn user if absolute path found in pom.xml"
```

```suggestion
	"description": "Code Transformation: Warn user if absolute paths are found in pom.xml"
```
Changelog entries are user-facing and listed right on the marketplace pages, so they need to be more friendly than commit style messages